examples/python-telegram-bot/Procfile

```diff
@@ -1 +1 @@
-web: python bot.py
\ No newline at end of file
+worker: python3 bot.py
```

Comment:
This is a bad change. It has to be `web`.

Reply:
Oo i see